scripts/ctest_parser.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
+"""
+ Copyright (C) 2022 Intel Corporation
+
+ Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+ See LICENSE.TXT
+ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""
+
+from subprocess import  Popen, DEVNULL
+import argparse
+import os
+import json
+
+TMP_RESULTS_FILE = "tmp-results-file.json"
+CTS_TEST_SUITES = ["context", "device", "enqueue", "event", "kernel", "memory",
```

Comment:
I was populating a list like this automatically when writing the script to save needing to maintain the list for any future changes. You can first run `ctest --show-only` in ` build/test/conformance` and it'll give you a list of each test suite. You can also pass `-V`and you can get the full path to the executable too.

Reply:
Thanks - I've added this to the script